src/index.js

```diff
@@ -344,7 +356,8 @@ class Dropzone extends React.Component {
       style: { display: 'none' },
       multiple: supportMultiple && multiple,
       ref: this.setRefs,
-      onChange: this.onDrop
+      onChange: this.onDrop,
+      disabled: disableDropzone
```

Comment:
After the prop is renamed, let's put it after the `accept` one using the shorthand notation.